### PR TITLE
Support passing table meta information over cqproto

### PR DESCRIPTION
**Pull request by roneli**

### CODE REVIEW

File: cqproto/provider.go [hunk 1]
```diff
@@ -5,6 +5,7 @@ import (
 	"context"
 
 	"github.com/cloudquery/cq-provider-sdk/provider/schema"
+
```
Comment (disq) [inline cqproto/provider.go:8]: stray newline

File: provider/schema/column.go [hunk 2]
```diff
@@ -275,3 +278,38 @@ func (c Column) checkType(v interface{}) bool {
 
 	return false
 }
+
+func (c Column) GetMeta() *ColumnMeta {
```
Comment (disq) [inline provider/schema/column.go:282]: I would call this `Meta()` to make it [idiomatic](https://golang.org/doc/effective_go#Getters).

File: cqproto/grpc.go [hunk 3]
```diff
@@ -195,11 +195,11 @@ func tablesFromProto(in map[string]*internal.Table) map[string]*schema.Table {
 func tableFromProto(v *internal.Table) *schema.Table {
 	cols := make([]schema.Column, len(v.GetColumns()))
 	for i, c := range v.GetColumns() {
-		cols[i] = schema.Column{
+		cols[i] = schema.SetColumnMeta(schema.Column{
```
Comment (disq) [inline cqproto/grpc.go:198]: Looks convoluted,  maybe there's a simpler/cleaner way to convert between `internal.Column` and `schema.Column`? a method on `intenral.Column` that gives you `schema.Column` maybe? If import cycles won't be an issue that is

**Review by disq — CHANGES_REQUESTED** [review]
LGTM overall, with recommendations

File: cqproto/grpc.go (hunk 3 above)
Comment (roneli) [reply cqproto/grpc.go:198]: internal.Column is auto generated

**Review by disq — APPROVED** [review]

**Pull request merged**